common/network-common/src/test/java/org/apache/spark/network/RequestTimeoutIntegrationSuite.java

```diff
@@ -110,7 +110,7 @@ public StreamManager getStreamManager() {
       }
     };
 
-    context = new TransportContext(conf, handler);
+    context = new TransportContext(conf, handler, true);
```

Comment:
Which patter does Apache Spark use; (1) `new TransportContext(conf, handler, true)`, (2) `new TransportContext(conf, handler)`. Is this test coverage change valid?

Reply:
Spark has both usage pattern with the later has `closeIdleConnection` as `false`.  But the previous behavior ignored `closeIdleConnection`, so I think it's safe to update this test to match the correct behavior.

Reply:
Could you keep this comment `open` for a while?

Reply:
Sure